src/main/scala/com/typesafe/sbt/digest/SbtDigest.scala

```diff
@@ -14,7 +14,7 @@ object Import {
 
   object DigestKeys {
     val algorithms = SettingKey[Seq[String]]("digest-algorithms", "Types of checksum files to generate.")
-    val indexPath = TaskKey[Option[String]]("digest-index-path", "Path to the generated asset index file.")
+    val indexFile = TaskKey[Option[File]]("digest-index-file", "Generated asset index file.")
```

Comment:
For compatibility, could we still have `indexPath` and create the `indexFile` from it, if it's not explicitly defined?